qooxdoo-kit/Dockerfile

```diff
@@ -1,17 +1,31 @@
-FROM node:10-alpine
+FROM node:10.16.3-alpine
```

Comment:
maybe have the version in an ARG with a default to make it more explicit

```
ARG ALPINE_VERSION=10.16.3
FROM node:${ALPINE_VERSION}-alpine
```

Reply:
I see your point but I think this is overdoing a bit at this stage :-)

I find this useful to e.g. sync versions among base images within this repo or in a multistage build as explained in
https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact.

Reply:
whatever you mean by this ... the idea is about 'readable code' and implicit documentation ... as a casual reader might not get that this is the node version and not the version of alpine ... just an idea ...

Reply:
As I said, I understand your point and I take note. thx :-)